packages/botonic-plugin-contentful/src/nlp/node-nlp.ts

```diff
@@ -0,0 +1,13 @@
+//import NlpUtil from 'node-nlp/lib/nlp/nlp-util';
+import stemmer = require('node-nlp/lib/nlp/stemmers/natural/porter-stemmer-es');
```

Comment:
Is there a better way to import stemmer?

Reply:
looks like "import stemmer from 'node-nlp/lib/nlp/stemmers/natural/porter-stemmer-es';" also works. Good spot! fixed
If you meant about the long path, see my comment at the d.ts file:
"// We use directly the ES Stemmer instead of using NlpUtil to avoid loading dependencies of so many languages
// (there are even errors with 'fs' library and with Japanese dependencies)
"